verticapy/tests/vModel/test_delphi.py

```diff
@@ -54,6 +54,7 @@ def winequality_vd():
 
 
 class TestDelphi:
+    @pytest.mark.skip("skipping for now as this needs to be fixed")
     def test_AutoML(self, titanic_vd):
```

Comment:
You can add this detail to the PR description as well.

Reply:
done